include/frg/formatting.hpp

```diff
@@ -12,7 +12,7 @@
 #include <frg/string_stub.hpp>
 
 /* the ranges library is not even partially freestanding for some reason */
-#if __STDC_HOSTED__
+#if __STDC_HOSTED__ && !defined(__clang__)
```

Comment:
do we know what range of clangs this is, so we can constrain it better?

Reply:
AFAICT it is still the case with current master.